### DISCUSSION TITLE
建议增加一个一键复制回答的功能

### DISCUSSION THREAD

**Discussion by sscc9** [GaiZhenbiao/ChuanhuChatGPT#469] [Ideas]
还有就是这个脚本是不是有问题，好像很难安装上，不是提示缺少什么依赖，而是提示文件里没有某些东西，或者就是用镜像功能时，提示命令不是函数还是什么的

**Comment by Keldos-Li** [comment]
1. 复制回答：有考虑，有思路，有实践，（可能我负责，）但最近可能没空做；

2. 因为现在包的依赖关系太复杂了，不同版本更新的很快，然后又相互要求不同版本，可能有一定问题。请尽量用高版本的python安装。
